### ISSUE TITLE
zabbix-agent includes could be used multiple times

### ISSUE THREAD

**Issue by aboe76** [saltstack-formulas/zabbix-formula#28]
Is it possible to extend zabbix-agent conf to have multple includes via pillar

pillar example:

```
include:
  - /etc/zabbix/zabbix_agentd.userparams.conf
  - /etc/zabbix/zabbix_agentd.d/
  - /etc/zabbix/zabbix-agent-extensions.d/
```

**Comment by michaelforge** [comment]
@aboe76 
Not at the moment, but it must be easy to extend `/files/default/etc/zabbix/zabbix_agentd.conf.jinja` to support lists from the pillar. `userparameters` pillar implementation could be used as reference.

**Comment by aboe76** [comment]
@michaelforge, I know this even works:
```
extra_conf: |
      include: /etc/zabbix/zabbix-agent-extensions.d/
```

but I want to make it as clean as possible with backward compatibility.. that's why I want to discuss it first..

**Comment by hatifnatt** [comment]
This solved in #71 particularly [here](https://github.com/saltstack-formulas/zabbix-formula/blob/b99466a1c040e34bcfec5eab5a1a0845a5e9d526/zabbix/files/default/etc/zabbix/zabbix_agentd.conf.jinja#L304).
I think this issue can be closed.

**Comment by pauldalewilliams** [comment]
I agree with @hatifnatt.  The current jinja template supports a single include or multiples as given in the  comment that opened this issue.
